From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jami Kettunen <jami.kettunen@protonmail.com>
Date: Wed, 7 Oct 2020 23:06:21 +0300
Subject: [HACK] scripts: Stop appending "+" to localversion

This always gets added and doesn't look nice in the version string, so
it's purely for aesthetic reasons :p
---
 scripts/setlocalversion | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index af4754a35e66..a42f4008a9dc 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -139,7 +139,7 @@ elif [ "${LOCALVERSION+set}" != "set" ]; then
 	# If the variable LOCALVERSION is set (including being set
 	# to an empty string), we don't want to append a plus sign.
 	scm=$(scm_version --short)
-	res="$res${scm:++}"
+	#res="$res${scm:++}"
 fi
 
 echo "$res"
-- 
Armbian

